contracts/staking/ClusterRewards.sol

```diff
@@ -432,4 +448,19 @@ contract ClusterRewards is
     }
 
 //-------------------------------- User functions end --------------------------------//
+
+    bytes32 public constant RECEIVER_PAYMENTS_MANAGER = keccak256("RECEIVER_PAYMENTS_MANAGER");
+    
+    mapping(address => uint256) public receiverBalance;
+    mapping(address => uint256) public receiverRewardPerEpoch;
+
+    function _increaseReceiverBalance(address receiver, uint256 amount) onlyRole(RECEIVER_PAYMENTS_MANAGER) external override {
+        receiverBalance[receiver]+=amount;
+    }
+
+    function _setReceiverRewardPerEpoch(address signer, uint256 rewardPerEpoch) onlyRole(RECEIVER_PAYMENTS_MANAGER) external override {
+        address staker = receiverStaking.signerToStaker(msg.sender);
```

Comment:
it should be `signer` here instead of `msg.sender` because `msg.sender` will always be `rewardDelegators` contract.

Reply:
let's also figure out how this passed tests and what we can improve  on.

Reply:
btw should signer address be the one changing these limits because signer is supposed to be the address on the client and used to issue tickets and receiver address is the  one which manages all the config.